umpire/cache.py

```diff
@@ -11,6 +11,8 @@
 from . import unpack
 from maestro.tools.os_tools import check_pid
 from . import config
+import logging, sys
+# logger = logging.getLogger(__name__)
```

Comment:
Is there a reason you're not using logger?

Reply:
I expect he's just importing this cache.py module into somewhere else where the logger is set up

Reply:
nope, that should have been enabled, so good eyes